 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container hfurlan-db-1  Creating
 Container hfurlan-db-1  Created
 Container hfurlan-api02-1  Creating
 Container hfurlan-api01-1  Creating
 Container hfurlan-api01-1  Created
 Container hfurlan-api02-1  Created
 Container hfurlan-nginx-1  Creating
 Container hfurlan-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
api02-1 exited with code 0
api01-1 exited with code 1
nginx-1  | 2024/02/14 16:42:08 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.25.0.3:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/14 16:42:08 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.25.0.4:8080/clientes/1/extrato", host: "localhost:9999"
db-1     | running bootstrap script ... ok
nginx-1  | 2024/02/14 16:42:10 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/14 16:42:12 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1 exited with code 1
api02-1 exited with code 1
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1 exited with code 1
api02-1 exited with code 1
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1 exited with code 1
api02-1 exited with code 1
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1 exited with code 1
api02-1 exited with code 1
db-1     | performing post-bootstrap initialization ... ok
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | waiting for server to start....2024-02-14 16:42:14.674 UTC [49] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-14 16:42:14.687 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-14 16:42:14.718 UTC [52] LOG:  database system was shut down at 2024-02-14 16:42:13 UTC
db-1     | 2024-02-14 16:42:14.785 UTC [49] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
nginx-1  | 2024/02/14 16:42:14 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1  | Conectando no BD...
api01-1  | DB_HOSTNAME:db
api01-1  | DB_NAME:rinha
api01-1  | DB_USERNAME:admin
api01-1  | 2024/02/14 16:42:14 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:06 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:07 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:08 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:10 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:12 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api02-1  | Conectando no BD...
api02-1  | DB_HOSTNAME:db
api02-1  | DB_NAME:rinha
api02-1  | DB_USERNAME:admin
api02-1  | 2024/02/14 16:42:14 DB unreachable:dial tcp 172.25.0.2:5432: connect: connection refused
api01-1 exited with code 1
api02-1 exited with code 1
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/ddl.sql
db-1     | CREATE TABLE
db-1     | CREATE INDEX
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE FUNCTION
db-1     | CREATE FUNCTION
db-1     | CREATE TRIGGER
db-1     | CREATE TRIGGER
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/dml.sql
db-1     | INSERT 0 5
db-1     | 
db-1     | 
db-1     | waiting for server to shut down...2024-02-14 16:42:16.780 UTC [49] LOG:  received fast shutdown request
db-1     | .2024-02-14 16:42:16.904 UTC [49] LOG:  aborting any active transactions
nginx-1  | 2024/02/14 16:42:16 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db-1     | 2024-02-14 16:42:16.966 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
db-1     | 2024-02-14 16:42:16.968 UTC [50] LOG:  shutting down
db-1     | 2024-02-14 16:42:16.997 UTC [50] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-14 16:42:17.441 UTC [50] LOG:  checkpoint complete: wrote 941 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.348 s, sync=0.046 s, total=0.474 s; sync files=317, longest=0.012 s, average=0.001 s; distance=4315 kB, estimate=4315 kB; lsn=0/1921828, redo lsn=0/1921828
db-1     | 2024-02-14 16:42:17.466 UTC [49] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-14 16:42:17.668 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-14 16:42:17.669 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-14 16:42:17.669 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-14 16:42:17.698 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-14 16:42:17.727 UTC [69] LOG:  database system was shut down at 2024-02-14 16:42:17 UTC
db-1     | 2024-02-14 16:42:17.766 UTC [1] LOG:  database system is ready to accept connections
nginx-1  | 2024/02/14 16:42:18 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 172.25.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | Cliente 1 inserido no cache com valor de limite igual a 100000
api02-1  | Cliente 1 inserido no cache com valor de saldo inicial igual a 0
api01-1  | Cliente 1 inserido no cache com valor de limite igual a 100000
api01-1  | sql: no rows in result set
api02-1  | Cliente 3 inserido no cache com valor de limite igual a 1000000
api02-1  | Cliente 3 inserido no cache com valor de saldo inicial igual a 0
api02-1  | Cliente 2 inserido no cache com valor de limite igual a 80000
api02-1  | Cliente 2 inserido no cache com valor de saldo inicial igual a 0
api02-1  | Cliente 5 inserido no cache com valor de limite igual a 500000
api02-1  | Cliente 5 inserido no cache com valor de saldo inicial igual a 0
api02-1  | Cliente 4 inserido no cache com valor de limite igual a 10000000
db-1     | 2024-02-14 16:42:47.097 UTC [118] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:47.097 UTC [118] DETAIL:  Failing row contains (2, -81910, 80000).
db-1     | 2024-02-14 16:42:47.097 UTC [118] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:49.767 UTC [74] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:49.767 UTC [74] DETAIL:  Failing row contains (2, -86787, 80000).
db-1     | 2024-02-14 16:42:49.767 UTC [74] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:52.366 UTC [134] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:52.366 UTC [134] DETAIL:  Failing row contains (2, -81472, 80000).
db-1     | 2024-02-14 16:42:52.366 UTC [134] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:52.668 UTC [174] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:52.668 UTC [174] DETAIL:  Failing row contains (2, -81163, 80000).
db-1     | 2024-02-14 16:42:52.668 UTC [174] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:52.866 UTC [139] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:52.866 UTC [139] DETAIL:  Failing row contains (2, -81215, 80000).
db-1     | 2024-02-14 16:42:52.866 UTC [139] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:52.868 UTC [164] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:52.868 UTC [164] DETAIL:  Failing row contains (2, -80743, 80000).
db-1     | 2024-02-14 16:42:52.868 UTC [164] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:55.638 UTC [148] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:55.638 UTC [148] DETAIL:  Failing row contains (2, -80526, 80000).
db-1     | 2024-02-14 16:42:55.638 UTC [148] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:55.678 UTC [137] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:55.678 UTC [137] DETAIL:  Failing row contains (2, -81476, 80000).
db-1     | 2024-02-14 16:42:55.678 UTC [137] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:55.759 UTC [139] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:55.759 UTC [139] DETAIL:  Failing row contains (2, -84838, 80000).
db-1     | 2024-02-14 16:42:55.759 UTC [139] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:56.643 UTC [130] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:56.643 UTC [130] DETAIL:  Failing row contains (2, -86179, 80000).
db-1     | 2024-02-14 16:42:56.643 UTC [130] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
db-1     | 2024-02-14 16:42:56.643 UTC [157] ERROR:  new row for relation "saldos" violates check constraint "saldo_valido"
db-1     | 2024-02-14 16:42:56.643 UTC [157] DETAIL:  Failing row contains (2, -85644, 80000).
db-1     | 2024-02-14 16:42:56.643 UTC [157] STATEMENT:  with novo_saldo as (UPDATE saldos SET saldo = saldo - $1 WHERE cliente_id = $2 RETURNING saldo) insert into transacoes (cliente_id, valor, descricao, tipo, saldo) values ($3, $4, $5, $6, (select * from novo_saldo)) returning saldo
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:57 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
nginx-1  | 2024/02/14 16:42:58 [alert] 29#29: 1000 worker_connections are not enough
